lib/at_host/at_host.c

```diff
@@ -93,8 +93,8 @@ static void uart_rx_handler(uint8_t character)
 	/* Handle control characters */
 	switch (character) {
 	case 0x08: /* Backspace. */
-		/* Fall through. */
-	case 0x7F: /* DEL character */
+	case 0x7F: /* Fall through. */
+		/* DEL character */
```

Comment:
When there is nothing else in the case, I don't think the fall through comment is needed. Also, the comment of which character it is should be on the same line as the case, alternatively the comment below should say `Backspace and DEL character`.

```suggestion
	case 0x7F: /* DEL character */
```